aea/cli/scaffold.py

```diff
@@ -56,6 +57,50 @@
 from aea.helpers.ipfs.base import IPFSHashOnly
 
 
+COPYRIGHT_HEADER_TEMPLATE = """\
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright {year} Valory AG
```

Comment:
NOTE: author is still hardcoded

Reply:
Please open an issue for this we can address this on the next release

Reply:
It should be very easy to do this on this PR. the author can be retrieved from config and passed to this as an arg.